server/controllers/index.js

```diff
@@ -1,12 +1,19 @@
 const express = require("express");
-const { getLocations, addLocation } = require("./locations");
+const {
+  getLocations,
+  addLocation,
+  getSuperUserRecords,
+} = require("./locations");
+
 const { getReps, handleStreetRepRequest } = require("./reps");
 const admin = require("./admin");
+
 const router = express.Router();
 
 router.get("/locations", getLocations);
 router.post("/locations", addLocation);
 
+router.get("/reps/records", getSuperUserRecords);
```

Comment:
Sorry, one last change - can you make this /reps/locations to match the other locations endpoint?